force-app/main/default/lwc/updatePaymentMethodModal/updatePaymentMethodModal.js

```diff
@@ -1,108 +1,849 @@
-import { LightningElement, api, track } from 'lwc';
-import updatePaymentMethod from '@salesforce/label/c.updatePaymentMethod';
-import CreditCardPaymentMethod from '@salesforce/label/c.RD2_Credit_Card_Payment_Method_Label';
-import ACHPaymentMethodLabel from '@salesforce/label/c.RD2_ACH_Payment_Method_Label';
-import paymentMethod from '@salesforce/label/c.RD2_Payment_Method';
-import commonCancelAndClose from '@salesforce/label/c.commonCancelAndClose';
-import commonCancel from '@salesforce/label/c.commonCancel';
-
-const ESC_KEY_CODE = 27;
-const ESC_KEY_STRING = "Escape";
-const FOCUSABLE_ELEMENTS = "button";
-const TAB_KEY_CODE = 9;
-const TAB_KEY_STRING = "Tab";
+import { LightningElement, api, track, wire } from "lwc";
```

Comment:
This file is basically a copy of `rd2EntryForm.js` which I do not recommend. Is there a way to reuse the `rd2EntryForm` component with a few changes that can be sent as parameters or attributes to the component?

Reply:
Yes, we took some pieces of code to re-use, but let me re-use rd2EntryForm with some parameters.